sdk/python/packages/flet-core/src/flet_core/geolocator.py

```diff
@@ -58,12 +62,16 @@ def __init__(
         #
         ref: Optional[Ref] = None,
         data: Any = None,
+        on_position: OptionalEventCallable = None,
```

Comment:
Can you create a custom event callable as it is done here: https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-core/src/flet_core/markdown.py#L53-L57

This will prevent user from having to parse json data themselves.

Reply:
Done.